examples/server_readspeed.c

```diff
@@ -13,11 +13,20 @@
 #define UA_SERVER_H_
 #define UA_CONNECTION_H_
 #define UA_TYPES_GENERATED_H_
+#define UA_LOG_H_
+#define UA_SECURITYCONTEXT_H_
+#define UA_SECURITYPOLICY_H_
 #include "server/ua_services.h"
 #include "ua_types_encoding_binary.h"
 
 int main(int argc, char** argv) {
     UA_ServerConfig config = UA_ServerConfig_standard;
+
+    // Set desired security policies
+    UA_SecurityPolicy securityPolicies[] = { UA_SecurityPolicy_None };
```

Comment:
If we can avoid memory allocation in the config, that would be better.
But it seems we have reached a point where this is no longer possible.

For this PR, it would be enough if the config **contains** a pointer to allocated memory.
Something like this:
```c
UA_ServerConfig DefaultConfig_new(UA_ByteString serverCertificate, UA_ByteStringprivateKey);
void DefaultConfig_deleteMembers(UA_ServerConfig *config);
```

That's just an example. How this exactly looks like is left to you imagination.
At a later point, we can have DefaultConfig_new return a pointer itself.